compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeWindow.uikit.kt

```diff
@@ -88,10 +88,39 @@ internal actual class ComposeWindow : UIViewController {
     private val keyboardOverlapHeightState = mutableStateOf(0f)
     private val safeAreaState = mutableStateOf(IOSInsets())
     private val layoutMarginsState = mutableStateOf(IOSInsets())
+
+    /*
+     * Initial value is arbitarily chosen to avoid propagating invalid value logic
+     * It's never the case in real usage scenario to reflect that in type system
+     */
     private val interfaceOrientationState = mutableStateOf(
-        InterfaceOrientation.getStatusBarOrientation()
+        InterfaceOrientation.Portrait
     )
 
+    /*
+     * On newest iOS interfaceOrientation will be deduced from [UIWindowScene] of [UIWindow]
```

Comment:
> newest

Please specify versions. This comment will be there in a few years with new iOS and possible new API

Reply:
Done